File: Examples/test-suite/ruby/using_composition_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'using_composition'

include Using_composition

f = FooBar.new
if f.blah(3) != 3
  raise RuntimeError,"blah(int)"
end

if f.blah(3.5) != 3.5
  raise RuntimeError,"blah(double)"
end

if f.blah("hello") != "hello"
  raise RuntimeError,"blah(char *)"
end

File: Examples/test-suite/python/using_composition_runme.py
from using_composition import *

f = FooBar()
if f.blah(3) != 3:
    raise RuntimeError,"blah(int)"

if f.blah(3.5) != 3.5:
    raise RuntimeError,"blah(double)"

if f.blah("hello") != "hello":
    raise RuntimeError,"blah(char *)"
